### ISSUE TITLE
Change BOM to reflect RP400 & larger touchscreen?

### ISSUE THREAD

**Issue by juggler2000** [ISS-Mimic/Mimic#519]
I suggest we change our RasPi BOM to reflect use of the RP400 and an optional larger touchscreen or end-user's existing non-touchscreen monitor, as this has become our standard.

Currently $100 for the RP400 which includes built-in keyboard, mouse, mircoSD, power cable, USB cable, and HDMI (may need HDMI adapter depending on monitor/touchscreen port)
https://www.adafruit.com/product/4796

Existing BOM
![image](https://github.com/ISS-Mimic/Mimic/assets/24573838/788bf61a-2e7b-4d89-a530-8d571fc36de6)

**Comment by GodspeedDiscvry** [comment]
I agree adding it, but do not agree replacing the smaller screen.  BOM should reflect both options.  Also - should probably not add larger screen until code is fixed.
